tests/factories/item_factories.py

```diff
@@ -15,6 +15,7 @@ class ItemFactory(SQLAlchemyModelFactory):
     summary = 'Summary of the feed item.'
     text = 'Text of the feed item.'
     feed = factory.SubFactory(FeedFactory)
+    totalVotes = 0
```

Comment:
We should remove this line again. I don't think we need it since the factory will use the default of 0 on the item model.